amethyst_renderer/src/cam.rs

```diff
@@ -76,6 +79,22 @@ impl Camera {
             std::f32::consts::FRAC_PI_3,
         ))
     }
+
+    /// Transforms position from screen space to camera space
+    pub fn position_from_screen(
+        &self,
+        screen_position: (f64, f64),
+        camera_transform: &GlobalTransform,
+        screen_dimensions: &ScreenDimensions,
+    ) -> (f32, f32) {
+        let screen_x = 2.0 * screen_position.0 as f32 / screen_dimensions.width() - 1.0;
+        let screen_y = 1.0 - 2.0 * screen_position.1 as f32 / screen_dimensions.height();
+        let screen_point = Point3::new(screen_x, screen_y, 0.0).to_homogeneous();
```

Comment:
I'm not sure about 0.0, but it works fine for me

Reply:
What do you mean?

Reply:
I don't know if it's ok to set `0.0` for the z axis for `screen_point`

Reply:
I believe it is, considering Z information only resides in the camera. Could you try it with a weird camera, like one with translated and rotated?

Reply:
Yes, I tested it with parameters
```
transform.set_xyz(17.0, -2.0, 1.0);
transform.set_rotation_euler(0.0, 0.0, 1.0);
```
It works fine